substanced/folder/static/css/mimetype-icons.css

```diff
@@ -0,0 +1,392 @@
+/*
```

Comment:
I personally agree with substanced/folder having its own static, and in my opinion it serves a single purpose: modularization. There's been a previous effort to make substanced/folder be self-contained, in a way that it could even be separated out into its own package at one point. I am not arguing that this should or would ever happen, but from a modularization point of view, I agree that this self-containment is a good thing. In general, with any large project, keeping all different type of sources together according to components, is a better strategy than separating some type of sources and maintaining a non component related folder structure.

However please note that this decision is not the scope of this pull request, because substanced.folder already has its own static, and its js and css lives there. In this case I am just adding some resources to the existing ones, which in this case belong to substanced.folder's static rather than to the global static.